scout/server/blueprints/cases/views.py

```diff
@@ -748,6 +752,26 @@ def rerun(institute_id, case_name):
     return redirect(request.referrer)
 
 
+@cases_bp.route("/<institute_id>/<case_name>/reanalysis", methods=["POST"])
+def reanalysis(institute_id, case_name):
+    """Toggle a rerun by making a call to RERUNNER service."""
+    edited_metadata = json.loads(request.form["sample_metadata"])
+
+    try:
+        controllers.call_rerunner(store, institute_id, case_name, edited_metadata)
+    except ReadTimeout as err:
+        msg = f"Error processing request: {err.__class__.__name__}"
+        flash(msg, "danger")
+    except RerunnerError as err:
```

Comment:
It's looking like this now: 

![image](https://user-images.githubusercontent.com/28093618/120291474-a0f8c000-c2c3-11eb-9e58-66b06c166cf2.png)